weave/tests/trace_server_clickhouse_conftest.py

```diff
@@ -108,3 +118,107 @@ def server_healthy(num_retries=1):
 
     # wait for the server to start
     return server_healthy(num_retries=30)
+
+
+class TwoWayMapping:
```

Comment:
This entire block was taken from to `weave/conftest.py` so that we could share the converter logic